nginxconf/default.conf

```diff
@@ -17,8 +17,9 @@ server {
     server_name  localhost;
 
     #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  bodylog;
-    access_log  logs/host.access.log  bodylog;
+
+    access_log  syslog:server=172.20.0.4:9898,severity=info bodylog;
```

Comment:
Again - using IP here isn't a good choice. Rather access syslog service through its service name in docker-compose - this is guaranteed to work.